### DISCUSSION TITLE
Cpdf::ACROFORM_FIELD_TEXT how remove background color and border

### DISCUSSION THREAD

**Discussion by sagarprostartme** [dompdf/dompdf#3346] [Q&A] [answered]
Currently :  
![image](https://github.com/dompdf/dompdf/assets/117825762/f356dcde-e615-46b7-b9fb-44e5e92d6845)

Expected result
![image](https://github.com/dompdf/dompdf/assets/117825762/58505dc7-a506-4c44-b34a-cc9b48d31c50)

```
  $cpdf = $canvas->get_cpdf();
  $cpdf->addForm();
  $cpdf->setStrokeColor(\Dompdf\Css\Color::parse('white'));
  $cpdf->setColor(\Dompdf\Css\Color::parse('white'));
  $cpdf->rectangle($x1, $y1, $w, $h);
  $companyField = $cpdf->addFormField(Cpdf::ACROFORM_FIELD_TEXT, 'company_name', $x1, $y1, $x2, $y2);
```
  
```
  <td colspan="6">
        <div id="company_name" style="height: 36px;background-color: transparent"></div>
    </td>
```

**Comment by bsweeney** [comment, marked as answer]
Form field highlight is an acrobat feature, not something that Dompdf is doing. Users can control that feature through the app preferences.

I'd advise against overriding usability preferences, but if you feel that your field has sufficient styling to make it apparent that it is actually a fillable field you can try to disable the highlight using Acrobat JavaScript.

```
$cpdf->javascript("app.runtimeHighlight = false;");
```

I say try because JS may not be enabled by default in some viewers.
